sdk/src/wallet/message_interface/account_method.rs

```diff
@@ -254,7 +255,7 @@ pub enum AccountMethod {
     /// Expected response: [`Output`](crate::wallet::message_interface::Response::Output)
     #[serde(rename_all = "camelCase")]
     PrepareOutput {
-        params: OutputParamsDto,
+        params: Box<OutputParamsDto>,
```

Comment:
Why is this a box? Mayeb also make [iota-sdk](https://github.com/iotaledger/iota-sdk/tree/eba4f0b8988de8cb6c767e566b7f6a20b0940878)/[bindings](https://github.com/iotaledger/iota-sdk/tree/eba4f0b8988de8cb6c767e566b7f6a20b0940878/bindings)/[core](https://github.com/iotaledger/iota-sdk/tree/eba4f0b8988de8cb6c767e566b7f6a20b0940878/bindings/core)/[src](https://github.com/iotaledger/iota-sdk/tree/eba4f0b8988de8cb6c767e566b7f6a20b0940878/bindings/core/src)/[method](https://github.com/iotaledger/iota-sdk/tree/eba4f0b8988de8cb6c767e566b7f6a20b0940878/bindings/core/src/method)
/account.rs PrepareOutput a box

Reply:
Because the Hrp is now stack allocated, this makes the variant larger than others and clippy complains.